Multisig/UI/App/UIKit/AppSettingsViewController/ImportedKeyCell.swift

```diff
@@ -11,22 +11,18 @@ import UIKit
 class ImportedKeyCell: UITableViewCell {
     @IBOutlet private weak var addressInfoView: AddressInfoView!
 
-    var onRemove: (() -> Void)?
```

Comment:
how will remove work in this case?

Reply:
see the view controller - it will override the method

Reply:
👍